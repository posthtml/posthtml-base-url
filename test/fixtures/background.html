<body background="image.jpg">
  <table background="image.jpg">
    <tr>
      <th background="image.jpg">Header</th>
    </tr>
    <tr>
      <td background="image.jpg">Cell</td>
    </tr>
  </table>
</body>
