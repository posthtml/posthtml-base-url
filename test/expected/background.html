<body background="https://example.com/image.jpg">
  <table background="https://example.com/image.jpg">
    <tr>
      <th background="https://example.com/image.jpg">Header</th>
    </tr>
    <tr>
      <td background="https://example.com/image.jpg">Cell</td>
    </tr>
  </table>
</body>
